utils.js

```diff
@@ -89,7 +89,24 @@ exports.getAttrs = function (str, start, options) {
     }
     value += char_;
   }
-  return attrs;
+
+  if (options.allowedAttributes && options.allowedAttributes.length) {
+    return attrs.filter(function (a) {
+      let allowedAttributes = options.allowedAttributes;
+      let attr = a[0];
+      let allowed = false;
+      for (let i = 0; i < allowedAttributes.length; i++) {
```

Comment:
I would prefer [`includes`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes) or similar methods on `Array.prototype` to this loop.

Reply:
If it was just for matching strings, I would too, but how am I going to support regular expressions then?

Reply:
Use .some